src/agent/io/sourcemapper.ts

```diff
@@ -218,13 +225,12 @@ export class SourceMapper {
   }
 
   /**
-   * @param {string} inputPath The path to an input file that could possibly
-   *  be the input to a transpilation process.  The path should be relative to
-   *  the process's current working directory
    * @param {number} The line number in the input file where the line number is
    *   zero-based.
    * @param {number} (Optional) The column number in the line of the file
    *   specified where the column number is zero-based.
+   * @param {string} The entry of the source map info in this.infoMap.
```

Comment:
Please update this comment; the type isn't a string, and I'm not able to understand what this parameter should be based on this documentation.

Reply:
Done.